cmd/check_container.go

```diff
@@ -93,6 +93,12 @@ var checkContainerCmd = &cobra.Command{
 				cfg.EnabledChecks = engine.ScratchContainerPolicy()
 				cfg.Scratch = true
 			}
+
+			// if a partner sets `Host Level Access` in connect to `Privileged`, enable RootExceptionContainerPolicy checks
+			if certProject.Container.Privileged {
+				cfg.EnabledChecks = engine.RootExceptionContainerPolicy()
+				cfg.Privileged = true
```

Comment:
Related to my other comment, I see this is being set, but I'm not sure where it's being used.

Should we be setting this configuration and then letting the engine.NewForConfig function give us the right policy?

Reply:
We aren't using this value today, but maybe in the future it's need.

Reply:
Leave it out.